ns/init.go

```diff
@@ -0,0 +1,15 @@
+package ns
+
+var (
+	basePath string
```

Comment:
Why not initializing this to "/var/run/docker" ?
Other besides osl.namespace_linux may ask for this in future.
So we do not end in a situation where we may have to check on different clients they all use the right default path.